node/core/approval-voting/src/lib.rs

```diff
@@ -1785,17 +1830,25 @@ fn check_and_import_assignment(
 		assigned_candidate_hashes.push(assigned_candidate_hash);
 	}
 
+	let claimed_core_index = match assignment.cert.kind {
+		// TODO, near future: remove CoreIndex from certificates completely.
```

Comment:
Could you elaborate? Will this be addressed in the parent PR?

Reply:
Yeah, that should be a future PR, not parent. I'll log a separate issue for it.